js/jquery.keyboard.js

```diff
@@ -753,6 +753,11 @@ http://www.opensource.org/licenses/mit-license.php
 					k1 = k >= keyCodes.A && k <= keyCodes.Z,
 					k2 = k >= keyCodes.a && k <= keyCodes.z,
 					str = base.last.key = String.fromCharCode(k);
+				// check, that keypress wasn't rise by functional key
+				// space is first typing symbol in UTF8 table
+				if (k < keyCodes.space){ //see #549
+					return;
+				}
 				base.last.virtual = false;
 				base.last.event = e;
 				base.last.$key = []; // not a virtual keyboard key
```

Comment:
Hi! Thanks for figuring out how to get around this problem... I'd like to keep the `base.last` value up-to-date with every key event; please move the additions below the `base.last.$key = [];` line and make sure everything still works for you.